docs/mkdocs/docs/technical/contributing.md

```diff
@@ -1,19 +1,88 @@
-Setting up your OS dependencies
-===============================
+Docker Quickstart
+=================
 
-ManyLinux Docker image
-----------------------
-ManyLinux(https://github.com/pypa/manylinux) captures the common denominator of many Linux distributions,
-giving the confidence that code worked in this image runs everywhere.
+#### 1) Start the ArcticDB build docker image
```

Comment:
Does not apply to Windows/MacOS

Reply:
I know. Frankly don't think it's a build deal - this should be self evident. I can specify that this applies only to Linux.